examples/autoscaler-webhook/dockerfile

```diff
@@ -0,0 +1,24 @@
+# Gather dependencies and build the executable
+FROM golang:1.10.3 as builder
+
+WORKDIR /go/src/agones.dev
```

Comment:
Can I also grab a Makefile (you can copy the one from simple-udp) that sets up the right tag for the example image please? :smile: then I can push it up to gcr.io for you, and we know it has the right tag.

Reply:
Makefile added